src/server/controllers/verification/verify/stateless/verify.stateless.paths.yaml

```diff
@@ -62,78 +62,31 @@ paths:
                   type: string
       responses:
         "200":
-          description: The contract has been successfully checked or sourcified
+          description: Success
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  result:
-                    type: array
-                    items:
-                      type: object
-                      properties:
-                        address:
-                          type: string
-                          example: "0x00000000219ab540356cBB839Cbe05303d7705Fa"
-                        status:
-                          type: string
-                          example: "perfect"
-                        storageTimestamp:
-                          type: string
-                          format: date-time
-                          nullable: true
+                oneOf:
+                  - $ref: "../../errors.yaml#/components/schemas/ContractNoDeploy"
+                  - $ref: "../../errors.yaml#/components/schemas/ContractPerfect"
+                  - $ref: "../../errors.yaml#/components/schemas/ContractPartial"
```

Comment:
Are these errors under Success - 200?

Reply:
This is an old file... I don't know what's going on with this PR Review. You are looking at the previous commit